release notes/v0.43.0.md

```diff
@@ -0,0 +1,124 @@
+k6 v0.43.0 is here! :tada:
+
+Notable changes in this release include:
+- [xk6-browser](https://github.com/grafana/xk6-browser) is now bundled in k6 as an experimental module, and usable without a separate binary or compilation step!
+- Native support for JavaScript's `async`/`await`.
+- A new experimental module for distributed tracing.
+- Large refactoring of core components to simplify the code base, improve maintainability, and increase test coverage.
+- Bug fixes, UX improvements, and maintenance.
+
+Keep reading for the details.
+
+## Breaking changes
+
+- [#2807](https://github.com/grafana/k6/pull/2807) Use non-zero exit codes for tests aborted by Ctrl+C or the REST API.
+
+  Aborting a test run with Ctrl+C will now exit with code `105`, and stopping via the REST API will exit with code `103`.
+
+
+## New Features
+
+### xk6-browser is now a built-in module [#2884](https://github.com/grafana/k6/pull/2884)
+
+[xk6-browser](https://github.com/grafana/xk6-browser), a k6 extension that enables browser automation and end-to-end testing from k6 scripts, now comes bundled with k6 under the `k6/experimental/browser` import path. This makes it more accessible, as users no longer need to build their own binary with [xk6](https://github.com/grafana/xk6), or use a separate binary to run browser tests.
+
+All current xk6-browser scripts that work with v0.8.0 will continue to work with the built-in module in k6 v0.43.0, by simply changing the import path from `k6/x/browser` to `k6/experimental/browser`, and setting the environment variable `K6_BROWSER_ENABLED` to `true`. The environment variable is needed to minimize potential risks with launching browser or any other processes via k6 scripts. It's also a mechanism we use in the k6 Cloud, where browser tests are currently disabled.
```

Comment:
Hey @inancgumus, is this technically accurate? Are browser tests currently disabled in the Cloud, and we enable them on a per-customer basis, or they'll remain disabled for all public use?

Let me know if this needs clarification.

Reply:
Extending the question to @inancgumus. I'm not sure if mentioning our mechanism in the cloud is relevant here. But generally, the sentence that the browser tests are currently disabled by default is an important one and if we have a instructions how to sign-up to the beta testing link them here.

```suggestion
All current xk6-browser scripts that work with v0.8.0 will continue to work with the built-in module in k6 v0.43.0, by simply changing the import path from `k6/x/browser` to `k6/experimental/browser`, and setting the environment variable `K6_BROWSER_ENABLED` to `true`. The environment variable is needed to minimize potential risks with launching browser or any other processes via k6 scripts.
```

Reply:
Hey @imiric (and @olegbespalov), yes, it's technically accurate. The browser tests are currently **disabled** in the Cloud (including private (per-customer basis) and public). This first merge was about to run browser tests from the command line using the `k6` tool.

Reply:
Thanks for confirming @inancgumus.

@olegbespalov We do want to mention the status of browser tests in the Cloud. Is your concern about mentioning the specific way we block browser tests in the Cloud?

Reply:
Nope, not the concern exactly, but I was thinking that we could use these release notes as the opportunity to announce the beta testing program, but generally fine even with current statement.